src/ru/nsu/ccfit/zuev/osu/MainActivity.java

```diff
@@ -886,4 +887,10 @@ private boolean checkPermissions() {
             return false;
         }
     }
+
+    @Override
+    protected void onDestroy() {
+        NotificationManagerCompat.from(getApplicationContext()).cancelAll();
+        super.onDestroy();
+    }

```

Comment:
This can be removed. `onDestroy` is not called when the user exits the game from the multitask menu, which is the cause of this bug.

Reply:
We want to ensure the notification gets removed when the Activity is destroyed

Reply:
Gonna remove this later

Reply:
Let's just leave this as is as a preventive measure.